v03_pipeline/lib/annotations/enums.py

```diff
@@ -198,6 +198,7 @@
 CLINVAR_PATHOGENICITIES = [
     'Pathogenic',
     'Pathogenic/Likely_pathogenic',
+    'Pathogenic/Likely_pathogenic/Established_risk_allele',
```

Comment:
@hanars wanted to bring this new pathogenicity to your attention even though it is within seqr's pathogenic filtering range: https://github.com/broadinstitute/seqr/blob/master/hail_search/constants.py#L75

Reply:
yes should nto be an issue for seqr but appreciate the heads up!

Reply:
😬